docs/src-static/.htaccess

```diff
@@ -51,6 +51,14 @@ RedirectMatch 301 ^/docs/akka-http/([^/]+)/java.html$     https://doc.akka.io/li
 RedirectMatch 301 ^/docs/akka/2.5.(\d+)/scala.html$      https://doc.akka.io/libraries/akka-core/2.5/?language=scala
 RedirectMatch 301 ^/docs/akka/2.5.(\d+)/java.html$       https://doc.akka.io/libraries/akka-core/2.5/?language=java
 
+# ===================================
+# Akka CLI docs, https://github.com/akka/akka-sdk/pull/324
+# ===================================
+Redirect 301 /reference/cli/index.html                https://doc.akka.io/operations/cli/index.html
+Redirect 301 /reference/cli/installation.html         https://doc.akka.io/operations/cli/index.html
+Redirect 301 /reference/cli/using-cli.html            https://doc.akka.io/operations/cli/using-cli.html
+Redirect 301 /reference/cli/command-completion.html   https://doc.akka.io/operations/cli/command-completion.html
```

Comment:
We have removed `/reference/cli/index.adoc`? In that case, why does the old page exist on the server? When we rsync the docs, do we only add files to the server and never remove?

Reply:
I would usually use `rsync -avz --delete` to fully sync directories, to delete files that are no longer present in the source. Can also run with backup options (`--backup --backup-dir=...`), in case the source is broken for some reason.

Reply:
If we had used the `page-aliases` correctly, this wouldn't have been a problem as Antora generates html files doing the redirect to the new location.